activerecord/lib/active_record/test_fixtures.rb

```diff
@@ -25,6 +25,8 @@ def after_teardown # :nodoc:
       class_attribute :pre_loaded_fixtures, default: false
       class_attribute :lock_threads, default: true
       class_attribute :fixture_sets, default: {}
+
+      ActiveSupport.run_load_hooks(:active_record_fixtures, self)
```

Comment:
We need to document this new hook in `engines.md` https://github.com/rails/rails/blob/74554cde8b96f6b9ae3e0b31ffb37783d4f56d10/guides/source/engines.md#L1463

Reply:
I added it. Though.. the column header is "Class" and `TestFixtures` is is a mixin.. does it warrant it's section or explanation?

Reply:
I tried something more descriptive, let me know what you think.